.travis.yml

```diff
@@ -7,7 +7,7 @@ jdk:
 - oraclejdk8
 install: true
 script:
-- ./gradlew clean test
+- ./gradlew clean test -i
```

Comment:
Why bother with incremental? Travis runs the build from a fresh clone, in theory the `clean` is not even needed.